app/views/events/index.html.erb

```diff
@@ -1,10 +1,41 @@
 <p style="color: green"><%= notice %></p>
 
 <h1>Events</h1>
-<%= link_to "New event", new_event_path %>
+<%#= link_to "New event", new_event_path %>
+<%= link_to "New event", new_event_path(start_date: [@calendar[:day].label, @calendar[:month].label(format: '%m-%Y')].join('-')) %>
 
-<div id="events" class="space-y-4">
-  <% @events.each do |event| %>
-    <%= render event %>
+<div>
+  <% if params[:skip] %>
+    <%= link_to 'Show Calendar', events_path %>
+  <% else %>
+    <%= link_to 'Hide Calendar', events_path(skip: true) %>
+    <%= link_to 'Today', pagy_url_for(@calendar.send(:last_object_at, Time.zone.now), 1) %>
+    <%#= link_to 'Today (does not work)', pagy_calendar_url_at(@calendar, Time.zone.now) %>
```

Comment:
<img width="678" alt="Screenshot 2024-01-06 at 12 57 59" src="https://github.com/corsego/pagy-calendar/assets/13472945/f49188c7-03b1-42a8-95a7-24681c7fbef7">

Reply:
ooops... it is prepended to the Backend instead of the Frontend, hence not available in the view unless you use `helper_method`. The [dev](https://github.com/ddnexus/pagy/tree/dev) branch has the fix.

Reply:
oh indeed, this looks really cool https://github.com/ddnexus/pagy/commit/9f93dd2cd1f370e25ff14d75e19710c59507dc20# and solves a lot!